iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -401,8 +401,8 @@ BusExt getConnectableBus(int node) {
             // if nothing found, just take the first bus
             if (connectableBus2[0] == null) {
                 Iterator<CalculatedBus> it = getBuses().iterator();
```

Comment:
I don't remember why the original check is done using iterator but as getBuses() returns a Collection it would be more clear to use isEmpty()

Reply:
agree on using `isEmpty`